### DISCUSSION TITLE
way to check if the user already loged in

### DISCUSSION THREAD

**Discussion by derK0ch** [GodotNuts/GodotFirebase#328] [Q&A]
Hey, I need a way to Check if the user alredy loged in on an other device to ensure that no data is overwritten. Is there a way to do this?

**Comment by BearDooks** [comment]
@derK0ch I don't know if that is currently possible via the RestAPI. In terms of the RTDB this wouldn't be an issue, if you are using Firestore, you might need to work in some logic to make sure the data is more current than what is in there. Maybe adding a timestamp as a field and checking that before you write data
